create-rust-app_cli/src/content/project.rs

```diff
@@ -379,3 +380,15 @@ pub fn create_resource(backend: BackendFramework, resource_name: &str) -> Result
 
     Ok(())
 }
+
+pub fn check_cli_version() -> Result<()>{
+  let name = env!("CARGO_PKG_NAME");
+  let version = env!("CARGO_PKG_VERSION");
+  let informer = update_informer::new(registry::Crates, name, version);
+  logger::message(&format!("Welcome to use {} v{}", name, version));
+  if let Some(version) = informer.check_version().ok().flatten()  {
+    logger::message(&format!("New version {} is available!", version));
+    logger::message(&format!("If you want to update, try: {}", style("cargo install --force create-rust-app_cli").yellow()));
+  }
```

Comment:
Minor suggestion for you!

I tried running `create-react-app` and noticed this message:

![image](https://user-images.githubusercontent.com/4259838/163916912-67dd7a2f-c726-4377-90c2-4a618c6911cc.png)

What do you think about something like this?

```suggestion
  if let Some(new_version) = informer.check_version().ok().flatten()  {
    logger::message(&format!("You are running `{name}` {version}, which is behind the latest release ({new_version})."));
    logger::message(&format!("If you want to update, try: {}", style("cargo install --force create-rust-app_cli").yellow()));
  } else {
    logger::message(&format!("You are running the latest version of {name} ({version})!"));
  }
```

Reply:
it looks great, I make a update soon 😆